src/Autocomplete.js

```diff
@@ -123,20 +200,39 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
+	 * Adds the CSS class to mark that item as selected.
+	 * @protected
+	 */
+	selectItem_(option) {
+		dom.addClasses(option, 'active');
+	}
+
+	/**
 	 * Emits a `select` event with the information about the selected item and
-	 * hides the element.
-	 * @param {!Element} item The list selected item.
+	 * hides the list element.
+	 * @param {!Object} item The list selected item.
 	 * @protected
 	 */
-	onListItemSelected_(item) {
-		var selectedIndex = parseInt(item.getAttribute('data-index'), 10);
+	selectOption_(selectedItem) {
```

Comment:
It's a bit confusing to have both a `selectItem_` and a `selectOption_` function, since the names make it seem like the do the same thing.  Maybe instead of `selectItem_` it should be `activateItem_`, so that we can use two different concepts for these things (activating when we're just adding the `active` css class, and selecting when we're actually choosing that as the selected item);